src/utils/config.ts

```diff
@@ -50,7 +52,8 @@ export const config: TRuntimeConfig = {
     chainId: '23295',
     subgraph:
       'https://v4.subgraph.sapphire-testnet.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph',
-    websocketURL: 'https://test.websocket.predictoor.ai',
+    websocketURL: 'https://websocket.predictoor.ai',
```

Comment:
shouldn't this be `https://test.websocket.predictoor.ai`?

Reply:
There is no https://test.websocket.predictoor.ai for now

Reply:
But there should be, if there isn't than we can just keep as it is. If we change the endpoint, the predictions for websocket are not working because we are getting them for main network by using `https://websocket.predictoor.ai`

Reply:
okay so should it be null for now? what is your idea

Reply:
the websocketURL should be the same `https://test.websocket.predictoor.ai` and for `statisticsURL` we could keep either one. I suggest the test url because probably that is where the data is going to be deployed